app/models/article.rb

```diff
@@ -1,2 +1,5 @@
 class Article < ApplicationRecord
+    def is_available
+      Time.now >= publish_on
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.